lib/Calendar.js

```diff
@@ -91,7 +91,11 @@ function createCalendar(request) {
 
             return request(this.config, "REPORT", 1, xmlRequest)
                 .then(xmlParser.parseEvents)
-                .then(events => events.filter(event => moment(event.data.start).isSameOrAfter(start, "day")));
+                .then(events => {
+                    return events.filter(event => {
+                        return !event.data.type.recurring || moment(event.data.start).isSameOrAfter(start, "day");
```

Comment:
I don't know, maybe it is just me, but I would prefer:

```javascript
const isRecurring = event.data.type.recurring;
const isSameDayOrAfter = moment(event.data.start).isSameOrAfter(start, "day");

// maybe also in that case a const isNotRecurring would be ok
return (isRecurring === false) || isSameDayOrAfter;
```

What do you think?